app/ide-desktop/lib/client/src/project-management.ts

```diff
@@ -0,0 +1,250 @@
+/** @file This module contains functions for importing projects into the Project Manager.
+ *
+ * Eventually this module should be replaced with a new Project Manager API that supports importing projects.
+ * For now, we basically do the following:
+ * - if the project is already in the Project Manager's location, we just open it;
+ * - if the project is in a different location, we copy it to the Project Manager's location and open it.
+ * - if the project is a bundle, we extract it to the Project Manager's location and open it.
+ **/
+
+import path from 'node:path'
+import fss from 'node:fs'
+import electron from 'electron'
+import crypto from 'node:crypto'
+import fsSync from 'node:fs'
+import * as tar from 'tar'
+
+import { isProjectRoot, projectMetadataRelative } from './paths'
+import { getCommonPrefix } from '../../../utils'
+import { logger } from 'enso-content-config'
+import { BUNDLED_PROJECT_EXTENSION } from '../file-associations'
+import { PRODUCT_NAME } from 'enso-common'
+
+// =======================
+// === Project Import ===
```

Comment:
lines not aligned (too many `=` in first and third line).

Reply:
Fixed.